src/ucp/core/ucp_ep.c

```diff
@@ -1171,7 +1210,8 @@ void ucp_ep_config_init(ucp_worker_h worker, ucp_ep_config_t *config)
 
             if (config->key.am_lane != UCP_NULL_LANE) {
                 /* Must have active messages for using rendezvous */
-                ucp_ep_config_set_rndv_thresh(worker, config, lane,
+                tag_lanes[0] = lane;
```

Comment:
can we avoid this?

Reply:
ucp_ep_config_set_rndv_thresh gets array of BW lanes, this is just creation of array from single element